app/src/main/res/navigation/nav_graph.xml

```diff
@@ -40,22 +40,23 @@
 
     <fragment
         android:id="@+id/lastFragment"
-        android:name="mx.yellowme.youst.presentation.challenges.navigation.LastFragment"
+        android:name="mx.yellowme.youst.challenges.navigation.LastFragment"
         android:label="DestinationThreeFragment"
         tools:layout="@layout/screen_destination_three">
 
         <argument
             android:name="nameArg"
             app:argType="string" />
         <action
```

Comment:
Just to follow our xml files spacing convention, I think we could add a space here 🤔

Reply:
Nice catch! Thanks!